.github/workflows/ort.yml

```diff
@@ -3,19 +3,91 @@ name: ORT
 on:
   workflow_dispatch:
 
-env:
-  ORT_IMAGE: ghcr.io/oss-review-toolkit/ort-minimal
-
 jobs:
   ort:
     name: Run ORT
     runs-on: ubuntu-22.04
     steps:
     - name: Checkout Repository
       uses: actions/checkout@v4
+      with:
+        path: ort-server
+
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v4
+      with:
+        version: 9
+
+    - name: Setup Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - name: Get latest ORT version
+      run: |
+        echo "ORT_VERSION=$(curl -s https://api.github.com/repos/oss-review-toolkit/ort/releases/latest | jq -r .tag_name)" >> $GITHUB_ENV
+
+    - name: Install ORT
+      run: |
+        curl -L -o ort.tar.gz https://github.com/oss-review-toolkit/ort/releases/download/${{ env.ORT_VERSION }}/ort-${{ env.ORT_VERSION }}.tgz
+        tar xfz ort.tar.gz
+        rm ort.tar.gz
+        echo "ort-${{ env.ORT_VERSION }}/bin" >> $GITHUB_PATH
+
+    - name: Cache ORT Cache Directory
+      uses: actions/cache@v4
+      with:
+        path: ~/.ort/cache
+        key: ${{ runner.os }}-ort-cache
+
+    - name: Run ORT Analyzer
```

Comment:
Should we run `ort requirements --list=commands` beforehand for informational purposes?

Reply:
I removed it because I didn't see value in it.

Reply:
I agree that it probably doesn't help with ORT Server specifically. But in general it helps to understand which ORT requirements are already fulfilled by the default action runners, and which additional setup actions would be needed to fulfill all of ORT's requirements.

But I'm fine with not doing that here.